### ISSUE TITLE
Build on travis CI failed

### ISSUE THREAD

**Issue by veggiemonk** [dkhamsing/awesome_bot#72]
Hello @dkhamsing 

Thank you for awesome bot. It is quite useful for CI.

Although, when run against awesome-docker, Travis CI failed.

Here is the error:
```
> White listed:
/home/travis/.rvm/gems/ruby-2.2.0/gems/awesome_bot-1.8.1/lib/awesome_bot/cli.rb:47:in `sort_by': comparison of Fixnum with Array failed (ArgumentError)
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/awesome_bot-1.8.1/lib/awesome_bot/cli.rb:47:in `order_by_loc'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/awesome_bot-1.8.1/lib/awesome_bot/cli.rb:156:in `cli'
	from /home/travis/.rvm/gems/ruby-2.2.0/gems/awesome_bot-1.8.1/bin/awesome_bot:6:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-2.2.0/bin/awesome_bot:23:in `load'
	from /home/travis/.rvm/gems/ruby-2.2.0/bin/awesome_bot:23:in `<main>'
	from /home/travis/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `eval'
	from /home/travis/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `<main>'
```

And here is the `.travis.yml`

```
language: ruby
rvm:
  - 2.2
before_script:
  - gem install awesome_bot
script:
  - awesome_bot README.md --white-list wiki/Installing,slideshare,awesome-docker.svg,edit/master,sindresorhus/awesome,ashmckenzie,shortcircuit.net.au,photo777.org,adrianmouat,redhat,viget,meros,toedter,datadoghq,amazon.cn,blog.codeship.com
```
Can you tell me what cause this error?

Thank you

**Comment by dkhamsing** [comment]
Will look into it!

**Comment by dkhamsing** [comment]
@veggiemonk found the issue! should be fixed shortly 😅

**Comment by dkhamsing** [comment]
This is fixed in version `1.8.2`
